### DISCUSSION TITLE
Which Altitude Reference is Used by Gpredict?

### DISCUSSION THREAD

**Discussion by AlexLandherr** [csete/gpredict#345] [Q&A] [answered]
I'm wondering which altitude value for the ground station location I should enter?

I've taken my glossary of terms from here:
[Elevation for beginners](http://eos-gnss.com/knowledge-base/articles/elevation-for-beginners)

Elevation types:
1. Ellipsoidal Height
2. Geoid Height
3. Orthometric Height

I've tried looking at the user manual but found nothing definitive.

**Comment by mdblack98** [comment, marked as answer]
I would think ellipsoidal as that's what satellites use
https://gssc.esa.int/navipedia/index.php/Reference_Frames_in_GNSS
 

    On Wednesday, January 17, 2024 at 07:19:15 AM CST, Alex Landherr ***@***.***> wrote:  
 
 


I'm wondering which altitude value for the ground station location I should enter?

I've taken my glossary of terms from here:
Elevation for beginners

Elevation types:
   
   - Ellipsoidal Height
   - Geoid Height
   - Orthometric Height

I've tried looking at the user manual but found nothing definitive.

**Reply by AlexLandherr** [reply]
Thanks! Was just curious about it since my future ground station needs accurate position data.

**Comment by wkitty42** [comment]
i've only ever used the standard ASL (Above Sea Level) like seen on most digital map apps and devices that have GPS capabilities... it will get you close enough... just be sure to use the proper unit measure for the software :wink: 
eg: 280ft ASL is 85.34meters ASL... round as needed...
